_config.yml

```diff
@@ -26,7 +26,7 @@ logoalt: 18F logo
 # Navigation
 # List links that should appear in the site sidebar here
 navigation:
-- text: The 18F Partnership Playbook
+- text: 18F Delivery: Partnership Playbook
```

Comment:
When you've got a string that contains a `:`, you need to wrap it in quotation marks.